sqlitecluster/SQLite.cpp

```diff
@@ -1201,6 +1208,16 @@ void SQLite::SharedData::commitTransactionInfo(uint64_t commitID) {
     _committedTransactions.insert(_preparedTransactions.extract(commitID));
 }
 
+void SQLite::SharedData::incrementOpenTransactions() {
+    lock_guard<decltype(_internalStateMutex)> lock(_internalStateMutex);
+    openTransactionCount++;
+}
+
+void SQLite::SharedData::decrementOpenTransactions() {
+    lock_guard<decltype(_internalStateMutex)> lock(_internalStateMutex);
```

Comment:
if this is atomic, you don't need to use lock_guard, right?

Reply:
+1, which would mean both methods are not necessary

Reply:
good catch, I found that we were doing that here, so did it to be safe
https://github.com/Expensify/Bedrock/blob/5211d06dc554851360c8a583467c02aa416471b5/sqlitecluster/SQLite.cpp#L1184-L1192

`atomic<bool> _commitEnabled;` is the declaration for this

Reply:
The above isn't done to protect `_commitEnabled`, it's done to prevent us from changing `_commitEnabled` while some other thread is committing.